app/lib/fact_check_email_handler.rb

```diff
@@ -9,8 +9,6 @@
 class FactCheckEmailHandler
   attr_accessor :fact_check_config
 
-  class UnableToProcessError < StandardError; end
```

Comment:
If we decide to log it (see my other comment) I think it would be good to keep this exception around so we get a nicer log message and something to search for.